### ISSUE TITLE
[Overview] Setup chapter

### ISSUE THREAD

**Issue by bartvanb** [vantage6/vantage6-workshop#48]
**Desired solution**

TODO:
- [x] Update version 4.7.x to the final version once we have tested everything
- [x] Somewhere there should be a sentence who to contact in case of issues. Preferably both at the start and at the end.
- [x] We should make it clear that the setup only has to be done for those participating in day 2 - for first day it's only UI right?
- [x] In addition to @bartvanb comments to the section only being applicable for people attending day 2, I would also mention the relevant chapters (in case someone does the course without us)
**Software Setup**
- [x] In the requirements list. 
  - [x] Mention that the Docker engine is preferred for Linux. I have no experience with DD for Linux and the engine is sufficient as we are not using any of the Desktop specific features. (also should increase the performance as Docker Desktop always runs in a VM)
  - [x] Mention miniconda (this is also done in the Conda section)
  - [x] Mention that they need a code editor preferable VSCode or PyCharm
 
Discuss:
**Docker**
- [x] I would prefer to let them run `docker run hello-world` as that would also catch Linux users who have not done the post-installation steps
**Python environment and Python packages**
- [x] We probably should mention the vantage6 version here as we are probably ahead when the workshop starts.

**Comment by frankcorneliusmartin** [comment]
Comments addressed in: https://github.com/vantage6/vantage6-workshop/pull/61

Left one open to change the final version. So do **not** close this issue when merged.

**Comment by frankcorneliusmartin** [comment]
This should be merged before the mail for the participants is send #54

**Comment by bartvanb** [comment]
All tasks from the list have been done, however I just linked another task in the PR. If that is merged, this issue can be closed.
